### PR TITLE
Add function to get context from chunk

### PR DESCRIPTION
**Pull request by johanpostema**
Add function zck_get_ctx to get context from chunk

### CODE REVIEW

File: include/zck.h.in [hunk 1]
```diff
@@ -185,6 +185,9 @@ char ZCK_PUBLIC_API *zck_get_range_char(zckCtx *zck, zckRange *range)
 /* Get file descriptor attached to zchunk context */
 int ZCK_PUBLIC_API zck_get_fd(zckCtx *zck)
     ZCK_WARN_UNUSED;
+/* Get context from zchunk */
```
Comment (jdieter) [inline include/zck.h.in:188]: For clarity in the function name and a clear comment, let's do the following:
```
/* Get context from chunk */
zckCtx ZCK_PUBLIC_API *zck_get_chunk_ctx(zckChunk *chunk)
    ZCK_WARN_UNUSED;
```

**Review by jdieter — CHANGES_REQUESTED** [review]

File: include/zck.h.in (hunk 1 above)
Comment (johanpostema) [reply include/zck.h.in:188]: Sounds excellent, thank you for the quick reply.
Please let me know if you want me to create a new pull request with the suggested change.

Comment (jdieter) [reply include/zck.h.in:188]: No need to do a new PR, just update this one, ideally amending this commit.

**Comment by jdieter** [comment]
Manually merged in ccb671a5f3177deb1f5bbe8b14fa289bfba49453

**Comment by jdieter** [comment]
I've just released 1.4.0 with both of your fixes.

**Pull request closed without merging**